### PR TITLE
Export both styled and unstyled components

### PR DESCRIPTION
**Pull request by alimpens**

### CODE REVIEW

File: packages/react/rollup.config.mjs [hunk 1]
```diff
@@ -39,6 +39,47 @@ export default [
...
+        globals: outputGlobals,
+      },
+      {
+        file: './unstyled/index.js',
+        format: 'esm',
+        sourcemap: true,
+        globals: outputGlobals,
+      },
+    ],
+    external: [/@babel\/runtime/, 'react-dom', 'react'],
```
Comment (hcvdhaar) [inline packages/react/rollup.config.mjs:82]: Duplicate?

**Comment by hcvdhaar** [comment]
Not sure about the name `unstyled`, would opt for something like `base` or `raw`.

File: packages/react/rollup.config.mjs (hunk 1 above)
Comment (hcvdhaar) [reply packages/react/rollup.config.mjs:82]: nevermind, not on the same level

**Comment by alimpens** [comment]
> Not sure about the name `unstyled`, would opt for something like `base` or `raw`.

I thought about this a bit. Names that are often used for unstyled components are 'headless', 'whitelabel' or 'primitive'. Those options all feel a bit cryptic to me though, 'unstyled' feels like a more accurate description of what it is. I'll start a discussion in Slack

**Review by dlnr — DISMISSED** [review]

**Pull request merged**